Makefile

```diff
@@ -289,9 +289,20 @@ validate-python-sdk:
 validate-client-java:
 	git diff --quiet -- clients/java || (echo "Modification verification failed! java client"; false)
 
+validate-python-wrapper:
+	sphinx-apidoc -o clients/python-wrapper/docs clients/python-wrapper/lakefs sphinx-apidoc --full -A 'Treeverse' -eq && git diff --quiet -- clients/python-wrapper || (echo 'Modification verification failed! python wrapper client'; false)
+
 # Run all validation/linting steps
 checks-validator: lint validate-fmt validate-proto validate-client-python validate-client-java validate-reference validate-mockgen validate-permissions-gen
 
+python-wrapper-lint:
+	pylint clients/python-wrapper/tests --rc=clients/python-wrapper/tests/.pylintrc
+	pylint clients/python-wrapper/lakefs --rc=clients/python-wrapper/lakefs/.pylintrc
+
+python-wrapper-gen-docs:
+	sphinx-build -b html clients/python-wrapper/docs clients/python-wrapper/_site/
+	sphinx-build -b html clients/python-wrapper/docs clients/python-wrapper/_site/$$(python clients/python-wrapper/setup.py --version)
```

Comment:
just to understand how it works - how sphinx understand the relative path under `_site` that container the version information?

Reply:
I don't thunk sphinx understands anything - I think it just uses relative links, that's why it works (deployed and tested)